test/user.spec.js

```diff
@@ -0,0 +1,188 @@
+import jwt from "jsonwebtoken";
+import axios from "axios";
+
+import utils from "./utils.js";
+import { apiUrl } from "../src/constants.js";
+import Userfront from "../src/index.js";
+import userConstructor from "../src/user.js";
+
+const { user } = Userfront;
```

Comment:
Please import into all the tests like this, so that we are testing the way it will be used by the admins.